.github/ISSUE_TEMPLATE/question-query-template.md

```diff
@@ -15,8 +15,8 @@ A clear explanation of why is this not a bug or a feature request?
 
 **Setup (please complete the following information if applicable):**
  - OS: [e.g. iOS]
- - IDE : [e.g. IntelliJ]
- - Version of the Library used
+ - IDE: [e.g. IntelliJ]
+ - Library/Libraries: [e.g. com.azure:azure-core:1.16.0 (groupId:artifactId:version)]
```

Comment:
Maybe use the format maven uses:

```xml
<dependency>
    <groupId>com.azure</groupId>
    <artifactId>azure-core</artifactId>
    <version>1.16.0</version>
</dependency>
```

Reply:
For now I think getting the artifact identifier should be sufficient, we can update this again when needed